freqtrade/freqtradebot.py

```diff
@@ -474,7 +474,7 @@ def check_and_call_adjust_trade_position(self, trade: Trade):
 
         if stake_amount is not None and stake_amount > 0.0:
             # We should increase our position
-            self.execute_entry(trade.pair, stake_amount, trade=trade)
+            self.(trade.pair, stake_amount, trade=trade)
```

Comment:
I assume that's not an intended change ... as i don't think that's valid python syntax ...